src/main/scala/com/databricks/spark/csv/CsvParser.scala

```diff
@@ -117,12 +116,12 @@ class CsvParser extends Serializable {
     this
   }
 
-  /** Returns a Schema RDD for the given CSV path. */
```

Comment:
This is just my personal thought. This refactoring looks pretty reasonable in a way though.. Could we maybe do this in a separate PR?